bundles/org.openhab.binding.telegram/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -25,7 +25,8 @@
 			</parameter>
 			<parameter name="chatIds" type="text" required="true" multiple="true">
 				<label>Chat Id(s)</label>
-				<description>One or more chat id(s). Access modifiers ("&lt;" for inbound only, "&gt;" for outbound only) can be used as prefix (optional).</description>
+				<description>One or more chat id(s). Access modifiers ("&lt;" for inbound only, "&gt;" for outbound only) can be
+					used as prefix (optional).</description>
```

Comment:
Do these line brakes happen when you run `make `or are they related to your editor? (same with pom.xml)

Reply:
This happens when i run mvn spotless:apply -pl :org.openhab.binding.telegram